### PR TITLE
Blog - History of NodeRED by Nick

### PR DESCRIPTION
**Pull request by Hasmin-AC**
## Related Issue(s)

WIP 

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/01/history-of-nodered.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+tags:
+    - posts
+    - flowfuse
+    
+---
+
+
+<!--more-->
+
+In January 2013, I could have never foreseen that my fun little proof-of-concept project would become Node-RED, an open source low-code environment with millions of deployments in IoT and automation.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:17]:
```suggestion
In January 2013, I could have never foreseen that my fun little proof-of-concept project would become Node-RED, an open source low-code environment with millions of deployments in IoT and automation.

<!--more-->
```

File: src/blog/2024/01/history-of-nodered.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 
+authors: 
+image: 
+tags:
+    - posts
+    - flowfuse
+
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:11]:
```suggestion
```

File: src/blog/2024/01/history-of-nodered.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 
+authors: 
+image:
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:7]: @Hasmin-AC Can you create a request for a social tile through the art request process? https://flowfuse.com/handbook/design/art-requests/#art-requests

File: src/blog/2024/01/history-of-nodered.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date:
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:5]:
```suggestion
date: 2024-02-08
```

File: src/blog/2024/01/history-of-nodered.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 
+authors:
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:6]:
```suggestion
authors: ["nick-oleary"]
```

File: src/blog/2024/01/history-of-nodered.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - flowfuse
+---
+
+
+<!--more-->
+
+In January 2013, I could have never foreseen that my fun little proof-of-concept project would become Node-RED, an open source low-code environment with millions of deployments in IoT and automation.
+
+Long before IoT became the ubiquitous term it is today, I was working in IBM’s Emerging Technology Group playing around with capturing data from devices and doing interesting things with it. The team focused on very fast-paced, short, proof-of-concept projects and was afforded time to learn new skills, innovate and work on side projects.
+
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/history-of-nodered.md:19]: @Hasmin-AC This will become a wall of text IMO, and hard to read. Do we need headings?

Comment (knolleary) [reply src/blog/2024/01/history-of-nodered.md:19]: @Hasmin-AC I'll give this a full review later today - there are a few tweaks I'd like to make to keep it more in my voice if that makes sense. I'll see what photos I can dig up that will be suitable.

File: src/blog/2024/02/history-of-nodered.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 2024-02-08
```
Comment (Hasmin-AC) [inline src/blog/2024/02/history-of-nodered.md:5]:
```suggestion
date: 2024-02-14
```

**Comment by ZJvandeWeg** [comment]
@Hasmin-AC whats needed to push this across the line?

**Comment by Hasmin-AC** [comment]
> @Hasmin-AC whats needed to push this across the line?

Nick wanted to make some edits and also add a few pictures.

File: src/blog/2024/01/history-of-nodered.md (hunk 6 above)
Comment (knolleary) [reply src/blog/2024/01/history-of-nodered.md]: Done - my proposed changes have now been merged into this branch

**Comment by knolleary** [comment]
This PR now contains my updates - and I've resolved the merge conflict.

It's good (from my pov) for final review and publish as/when its scheduled to go out.

**Review by knolleary — APPROVED** [review]

File: src/blog/2024/02/history-of-nodered.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 2024-02-14
```
Comment (Hasmin-AC) [inline src/blog/2024/02/history-of-nodered.md:5]:
```suggestion
date: 2024-02-28
```

File: src/blog/2024/02/history-of-nodered.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+authors: ["nick-oleary"]
+image: blog/2024/02/images/history-of-node-red.png
+tags:
+    - posts
+    - flowfuse
```
Comment (Hasmin-AC) [inline src/blog/2024/02/history-of-nodered.md:10]:
```suggestion
    - flowfuse
    - History of Node-RED
    - Node-RED
    -Node Red
```

**Pull request merged**